tests/bugfixes/github/test_issue_981.py

```diff
@@ -8,12 +8,12 @@ class CanonAfInfoTest(metaclass=CaseMeta):
     filenameB = path("$data_path/test_issue_981b.exv")
     filenameC = path("$data_path/test_issue_981c.exv")
     filenameD = path("$data_path/test_issue_981d.exv")
-    commands  = ["$exiv2 -pa -b --grep Canon.AF $filenameA",
-                 "$exiv2 -pa -b --grep Canon.AF $filenameB",
-                 "$exiv2 -pv -b --grep Points   $filenameC",
-                 "$exiv2 -pt -b --grep Points   $filenameC",
-                 "$exiv2 -pv -b --grep Unusable $filenameD",
-                 "$exiv2 -pt -b --grep Unusable $filenameD",
+    commands  = ["$exiv2 -pa -b --grep Canon.AF $filenameA"
+                ,"$exiv2 -pa -b --grep Canon.AF $filenameB"
+                ,"$exiv2 -pv -b --grep Points   $filenameC"
+                ,"$exiv2 -pt -b --grep Points   $filenameD"
+                ,"$exiv2 -pv -b --grep Unusable $filenameC"
+                ,"$exiv2 -pt -b --grep Unusable $filenameD"
```

Comment:
Why this change? Now we do `exiv2 -pv` twice for `filenameC` and `exiv2 -pt` twice for `filenameD` instead of one of these for each filename

Reply:
I think I made an error when I invented the test in 2019. So, I fixed it.  Files C and D are concerned with NumPoints and Unusbles.  I want to test both the metadata (-pv) and pretty printer (-pt).

Reply:
Crap you're right.  I've got the change wrong.  I'll fix that.